scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -501,6 +503,13 @@ def run_command(self, cmd, timeout):
                 self.status = TwisterStatus.FAIL
                 proc.kill()
 
+        if proc.returncode in (ExitCode.INTERRUPTED, ExitCode.USAGE_ERROR, ExitCode.INTERNAL_ERROR):
+            self.status = TwisterStatus.ERROR
+            self.instance.reason = f'Pytest failed with return code {proc.returncode}'
```

Comment:
IMO, this message suggests that the test result is more than expected. But in this case, it occurs error, which is related to the pytest environment and should be better highlighted.
```suggestion
            self.instance.reason = f'Pytest error - return code {proc.returncode}'
```

Reply:
updated